data/json/monsters/insect_spider.json

```diff
@@ -2969,5 +2969,379 @@
+    "bodytype": "flying insect",
+    "species": [ "INSECT_FLYING" ],
+    "default_faction": "insect",
+    "material": [ "iflesh" ],
+    "symbol": "L",
+    "color": "light_gray",
+    "volume": "70 L",
+    "weight": "40 kg",
+    "hp": 50,
+    "speed": 95,
+    "vision_night": 15,
+    "dodge": 2,
+    "harvest": "arachnid_firefly",
+    "biosignature": { "biosig_item": "insect_dust", "biosig_timer": 3 },
+    "reproduction": { "baby_egg": "egg_butterfly", "baby_count": 1, "baby_timer": 15 },
+    "baby_flags": [ "SPRING", "SUMMER" ],
+    "fear_triggers": [ "HURT", "FIRE" ],
+    "flags": [ "SEES", "HEARS", "FLIES", "STUMBLES" ]
+  },
+  {
+    "id": "mon_moth_titan",
+    "type": "MONSTER",
+    "name": { "str": "lunar moth" },
+    "description": "A majestic moth painted in pale colors.  This giant of the night only comes out after the moon is out, leaving glittering dust.",
+    "copy-from": "mon_moth_giant",
+    "proportional": { "hp": 3, "speed": 1.1, "morale": 1.1, "melee_dice_sides": 1.1 },
+    "symbol": "L",
+    "color": "cyan",
+    "volume": "120 L",
+    "weight": "75 kg"
+  },
+  {
+    "id": "mon_caterpillar",
+    "type": "MONSTER",
+    "name": { "str": "caterpillar" },
+    "description": "Caterpillars are the larval stage of butterflies and moths.  This one seems newly hatched from an egg but even now it is large as a cat.",
+    "default_faction": "ant",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "5 L",
+    "weight": "6 kg",
+    "hp": 10,
+    "material": [ "iflesh" ],
+    "symbol": "c",
+    "color": "green",
+    "speed": 50,
+    "vision_day": 15,
+    "dodge": 2,
+    "harvest": "arachnid",
+    "upgrades": { "age_grow": 15, "into_group": "GROUP_BUTTERFLY" },
+    "flags": [ "SEES", "SMELLS", "CLIMBS", "HEARS", "STUMBLES", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_caterpillar_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant caterpillar" },
+    "description": "This caterpillar has reached the size of a person and it looks close to make a cocoon.  You can only imagine the size of the insect that will emerge.",
+    "copy-from": "mon_caterpillar",
+    "proportional": { "hp": 3, "speed": 1.1, "morale": 1.1, "melee_dice_sides": 1.1 },
+    "symbol": "C",
+    "volume": "60 L",
+    "weight": "75 kg",
+    "upgrades": { "age_grow": 15, "into_group": "GROUP_BUTTERFLY_GIANT" }
+  },
+  {
+    "id": "mon_cicada_nymph",
+    "type": "MONSTER",
+    "name": { "str": "cicada nymph" },
+    "description": "The mutated larval form of a Cicada.  It doesn't produce noise like the adults but its digging claws seem suited for scratching.",
+    "default_faction": "insect",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "40 L",
+    "weight": "40 kg",
+    "hp": 80,
+    "speed": 80,
+    "material": [ "iflesh" ],
+    "symbol": "c",
+    "color": "white",
+    "aggression": -10,
```

Comment:
A bit too aggressive for "living in the ground, chewing roots".

Reply:
aggresiveness nerfed to -5 for both nymp and adult